build.gradle

```diff
@@ -39,4 +41,5 @@ allprojects {
 
 repositories {
     mavenCentral()
+    mavenLocal()
```

Comment:
Is this for the service. Or when we want to use the service on our machine>

Reply:
I was installing ab2d-libs locally while testing and added this so gradle would pick it up.